### PR TITLE
etcdserver: move unti out of server.go

### PR DESCRIPTION
**Pull request by xiang90**
etcdserver file is messy enough. Let's make it be less messy.

/cc @gyuho @heyitsanthony

### CODE REVIEW

File: etcdserver/util.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package etcdserver
+
+import (
+	"time"
+
+	"github.com/coreos/etcd/pkg/types"
+	"github.com/coreos/etcd/rafthttp"
+)
+
+// Copyright 2015 CoreOS, Inc.
```
Comment (gyuho) [inline etcdserver/util.go]: I think License header is reversed?

Comment (xiang90) [inline etcdserver/util.go]: sorry. fixed.

**Comment by gyuho** [comment]
LGTM

**Comment by heyitsanthony** [comment]
There's already a `cluster_util.go` in the etcdserver directory; should `util.go` be named `sever_util.go`, then?

**Comment by xiang90** [comment]
@heyitsanthony cluster_util.go is named as cluster_util.go since I wanted to group cluster related thing and move all cluster related stuff into pkg /etcdserver/cluster some day. But that cleanup has not happened until today. I guess change the name from util.go to server_util.go is not necessary since this is inside pkg etcdserver. But if you have a strong opinion, I would do the rename.

**Comment by heyitsanthony** [comment]
No strong opinion, just wondering about the inconsistency. LGTM

**Pull request merged**